packages/api/src/libs/osnap/interfaces.ts

```diff
@@ -0,0 +1,394 @@
+// osnap types imported from https://github.com/snapshot-labs/snapshot/blob/master/src/plugins/oSnap/types.ts
```

Comment:
this is literally copied and pasted from the osnap repo so we can get accurate space and transaction types ( what the osnap frontend sends this call)